x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/utils.ts

```diff
@@ -101,3 +103,28 @@ export function crawlDomainValidationToResult(
     state: 'valid',
   };
 }
+
+export const getDeleteDomainConfirmationMessage = (domainUrl: string) => {
```

Comment:
Do we want a unit test for this, like `getDeleteDomainSuccessMessage`?

Reply:
Yes we do